integration/tests/cook/test_basic.py

```diff
@@ -25,8 +25,8 @@ def setUp(self):
         util.wait_for_cook(self.cook_url)
 
     def test_basic_submit(self):
-        job_uuid, resp = util.submit_job(self.cook_url)
-        self.assertEqual(resp.status_code, 201)
+        job_uuid, resp = util.submit_job(self.cook_url, executor='cook')
```

Comment:
Modified the test instead and added a warning log.

Performing the validation requires threading an additional dependency 4-5 levels deep into `api/validate-and-munge-job`, so avoided it.